lib/cartopy/tests/crs/test_lambert_conformal.py

```diff
@@ -38,6 +39,17 @@ def test_default_with_cutoff():
                               (-49788019.81831982, 30793476.08487709))
 
 
+def test_sphere():
+    """Test LambertConformal with spherical globe. (#2377)"""
+    globe = ccrs.Globe(ellipse='sphere')
+
+    # This would error creating a boundary
+    crs = ccrs.LambertConformal(globe=globe)
+
+    assert all(np.isfinite(x) for x in crs.x_limits)
```

Comment:
Out of curiosity is there a reason to iterate instead of using `np.all()`?
```suggestion
    assert np.all(np.isfinite(crs.x_limits))
```

Reply:
For some reason my brain saw `x_limits` and `y_limits` were tuples and made me write out the iteration. 🙄 Fixed.